index.html

```diff
@@ -0,0 +1,64 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>John Thiong'o Murianki</title>
+    <link rel="stylesheet" href="styles.css">
+    <link href='https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap' rel='stylesheet'>
+</head>
+<body>
+    <div id ="content">
+    <div id ="status-bar"></div>
+    <header>
+        <a id="logo">
+            <img src="/media/My Logo.png">
+        </a>
+        <nav>
+            <div class="line"></div>
+            <div class="line"></div>
+            <div class="line"></div>
+        </nav>
+    </header>
+    <section id="greeting">
+        <p>I'm John Glad to see you</p>
```

Comment:
- [ ] As required before you should use a `<h1>` element here instead of using `<p>` because it will help search engines understand the meaning (context) of a page :+1:. [Check this link](https://www.reliablesoft.net/h1-tag/) for more info